packages/rendermime/src/renderers.ts

```diff
@@ -418,6 +418,12 @@ export function renderSVG(options: renderSVG.IRenderOptions): Promise<void> {
     return Promise.resolve(undefined);
   }
 
+  // Add missing SVG namespace (if actually missing)
+  var patt = '<svg[^>]+xmlns=[^>]+svg';
```

Comment:
Can you please change this to `const patt`?  The lint checker is failing on this line.